### PR TITLE
AER-2656 Added Monthly time varying emission profile to enum

### PR DESCRIPTION
**Pull request by Hilbrand**

### CODE REVIEW

**Review by BertScholten — APPROVED** [review]
LGTM

File: source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/CustomDiurnalVariationType.java [hunk 1]
```diff
@@ -43,7 +43,12 @@ public double sum(final List<Double> values) {
       return sumWeekdays + sumOtherDays;
     }
 
-  };
+  },
+
+  /**
+   * Monthly Time-varying emission profile over a year, with 1 value for each month.
+   */
+  MONTHLY(12);
```
Comment (BertScholten) [inline source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/CustomDiurnalVariationType.java:51]: Could be `MONTH` to be more in lin with `DAY`?

Comment (Hilbrand) [reply source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/CustomDiurnalVariationType.java:51]: I followed ADMS manual. This uses `3-day` and `monthly`. Therefore I guess monthly seems better.

Comment (BertScholten) [reply source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/CustomDiurnalVariationType.java:51]: In that case let's follow the native speakers I guess.

**Pull request merged**